TestsCalculator/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace TestsCalculator.Pages
+{
+    public class CalculatorPage
+    {
+        private IWebDriver _driver;
+
+        public CalculatorPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public IWebElement AmountField => _driver.FindElement(By.Id("amount"));
+        public IWebElement PercentField => _driver.FindElement(By.Id("percent"));
+        public IWebElement TermField => _driver.FindElement(By.Id("term"));
+        public IWebElement DayDropdown => _driver.FindElement(By.Id("day"));
+        public IWebElement MonthDropdown => _driver.FindElement(By.Id("month"));
+        public IWebElement YearDropdown => _driver.FindElement(By.Id("year"));
+        public IWebElement DaysRadioBtn365 => _driver.FindElement(By.Id("d365"));
+        public IWebElement DaysRadioBtn360 => _driver.FindElement(By.Id("d360"));
+        public IWebElement IncomeField => _driver.FindElement(By.Id("income"));
```

Comment:
If you use the field only to get its text, so you may declare it like here:
```suggestion
        public string Income => _driver.FindElement(By.Id("income")).GetAttribute("value");
```

And in tests you will write:
```
        Assert.AreEqual("1005.56", calculatorPage.Income);
```